### ISSUE TITLE
editor.clear() not clears prefs

### ISSUE THREAD

**Issue by xdigit** [yandextaxitech/binaryprefs#35]
Hi!
```
SharedPreferences.Editor editor = sharedPreferences.edit();
editor.clear();
editor.commit();
```

Not clears anything in my case, maybe some issues with threads, but calling editor.remove("some_string") for every preference do the job

**Comment by Yazon2006** [comment]
I can confirm that such problem exists. But I tried reproduce it on new clean project and unfortunatelly for me it worked well. Thats why I decided that problem is in my code but not in BinaryPreferences.

**Comment by iamironz** [comment]
Can you please ask what of version of binary preferences do you use?
Also one little question: can you reproduce this kind of issue on clear project?

**Comment by xdigit** [comment]
Version 1.0.0-BETA-2
I didn't try with clear project

**Comment by iamironz** [comment]
Should be fixed in the future release (1.0.0).
